velox/functions/sparksql/specialforms/AtLeastNNonNulls.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/functions/sparksql/specialforms/AtLeastNNonNulls.h"
+#include "velox/expression/SpecialForm.h"
+
+using namespace facebook::velox::exec;
+
+namespace facebook::velox::functions::sparksql {
+namespace {
+class AtLeastNNonNullsExpr : public SpecialForm {
+ public:
+  AtLeastNNonNullsExpr(
+      TypePtr type,
+      std::vector<ExprPtr>&& inputs,
+      bool inputsSupportFlatNoNullsFastPath)
+      : SpecialForm(
+            std::move(type),
+            std::move(inputs),
+            AtLeastNNonNullsCallToSpecialForm::kAtLeastNNonNulls,
+            inputsSupportFlatNoNullsFastPath,
+            false /* trackCpuUsage */) {}
```

Comment:
Why do we need to pass false, and shall we adapt to the 'trackCpuUsage' variable in constructSpecialForm?

Reply:
Updated, thanks.